### PR TITLE
CHAN-476 remove xpromo dismissal while app navigation

### PR DESCRIPTION
**Pull request by tomsoir**
Task: https://reddit.atlassian.net/browse/CHAN-476.
- also xpromo text color has been fixed in night mode.

### CODE REVIEW

**Comment by tomsoir** [comment]
:nail_care:

File: src/app/components/DualPartInterstitial/styles.less [hunk 1]
```diff
@@ -65,6 +65,9 @@
   &__headerPlaceholder {
     padding-bottom: 85%;
   }
+  &Footer{
+    color: @semi-black-text;
```
Comment (schwers) [inline src/app/components/DualPartInterstitial/styles.less]: This should live in `DualPartInterstitial/Footer/styles.less` instead. It looks like we're also not nightmode friendly at the moment for the general case interstitial.  

We could do something like
```diff
diff --git a/src/app/components/DualPartInterstitial/Footer/styles.less b/src/app/components/DualPartInterstitial/Footer/styles.less
index 7a04194..d502c11 100644
--- a/src/app/components/DualPartInterstitial/Footer/styles.less
+++ b/src/app/components/DualPartInterstitial/Footer/styles.less
@@ -10,9 +10,13 @@
   right: 0;
   bottom: 0;
   z-index: @z-index-navbar + 2;
-  background-color: @white;
   text-align: center;
 
+  .themeify({
+    background-color: @theme-body-color;
+    color: @theme-body-text-color;
+  });
+
   &__subtitle {
     font-weight: bold;
     font-size: 22px;
diff --git a/src/app/components/DualPartInterstitial/styles.less b/src/app/components/DualPartInterstitial/styles.less
index a88ca39..02ab603 100644
--- a/src/app/components/DualPartInterstitial/styles.less
+++ b/src/app/components/DualPartInterstitial/styles.less
@@ -65,9 +65,6 @@
   &__headerPlaceholder {
     padding-bottom: 85%;
   }
-  &Footer{
-    color: @semi-black-text;
-  }
 }
 
 .DualPartInterstitial + .TopNav {
```

**Comment by tomsoir** [comment]
@schwers 💇 :)

File: src/app/components/DualPartInterstitial/styles.less [hunk 2]
```diff
@@ -75,34 +90,4 @@
...
-      &__subtitle{
-        color: #fff;
-      }
-      &__bulletList{
-        display: none;
-      }
-    }
-  }
-}
+}
```
Comment (schwers) [inline src/app/components/DualPartInterstitial/styles.less]: Needs an extra newline for end-of-file

File: src/app/less/themes/themeify.less [hunk 3]
```diff
@@ -29,3 +29,15 @@
...
+  // apply default styles
+  & { @rules(); }
+  // apply custom theme styles (root rule will
+  // be ignored if current class is not exist)
+  &.xpromoMinimal,  
+   .xpromoMinimal & { 
+    @rules();
+    @minimal();
+  }
+}
```
Comment (schwers) [inline src/app/less/themes/themeify.less]: Needs an extra newline for end-of-file

Comment (tomsoir) [inline src/app/less/themes/themeify.less]: Fixed. Thank you :)

**Comment by tomsoir** [comment]
@schwers 💇 👍

**Pull request merged**